src/Client.ts

```diff
@@ -53,6 +55,7 @@ type ClientParams = {
   publicKey: string,
   targetEnvironment: string,
   targetProduct?: string,
+  targetVersion?: string
```

Comment:
I don't think we should do this. targetVersion if it is to be fetched by the SDK will change frequently. It shouldn't be part of the client initialization parameters.

Instead can we supply the argument to the function? @afzalsayed96